server/static/wrstat/src/Table.tsx

```diff
@@ -114,6 +116,12 @@ const noopFormatter = (a: { toString(): string }) => a + "",
 
 				onRowClick(row);
 			}}
+			onMouseOver={() => {
+				onRowHover(row);
```

Comment:
Is this spamming the debug console with errors if onRowHover and/or onMouseOut aren't set? Or have these been set for all tables?